Makefile

```diff
@@ -0,0 +1,61 @@
+COMPOSER_ARGS += --no-progress --no-interaction
```

Comment:
tohle by asi nemělo být forcnuté ... když to pustíš ručně, tak chce progress a interaction :D

Reply:
ale je to jen detail

Reply:
ja to kopiroval z nejakyho tvyho, asi odsud https://github.com/cdn77/coding-standard/blob/master/Makefile

Reply:
jasný no, myslel jsem si to :D problém je že s každým projektem se tohle postupně vylepšuje, takže si to imo vzal zrovna z toho staršího no :D nvm, je to prakticky jedno

Reply:
tak to pak nak updatni v tech projektech :D

Reply:
nebudu procházet všechny projekty co máme :D jak jsem psal, je to prakticky jedno, když to bude někoho srát, je možný to vždy upravit